### ISSUE TITLE
Can't Edit Oracle Driver Properties since 5.1.5

### ISSUE THREAD

**Issue by jluismoran** [dbeaver/dbeaver#3992]
Steps to reproduce:

- Right click in Oracle Connection > Edit Connection > Connection settings > Driver properties
- Can't Edit the value of the properties

It was working in 5.1.4 but today after installed new version I've noticed that now it is not possible, so I've turned back to previous version.

I'm using Linux Mint and I did the follow steps to install the new version:

- sudo apt-get remove dbeaver-ce
- sudo dpkg -i dbeaver-ce_5.1.5_amd64.deb

**Comment by serge-rider** [comment]
I can't reproduce this. What exactly is not working?
Properties are read-only or property list is empty or?..

**Comment by jluismoran** [comment]
Yes, the properties are displayed read-only. In version 5.1.4 I can edit the properties' value without any problem.

**Comment by serge-rider** [comment]
Weird. 
Maybe this is one of problems caused by migration to GTK3. could you check the workaround in #4002 (change dbeaver.ini)?

**Comment by Akint** [comment]
Same problem here.
Ubuntu 16.03, dbeaver-ce 5.1.5.
Workaround with `--launcher.GTK_version 2` helps, but causes some visual artifacts.

**Comment by serge-rider** [comment]
Please check version 5.1.6 (just released)

**Comment by Akint** [comment]
It works (driver properties are editable) without workarounds and artifacts in 5.1.6, thank you.

**Comment by jluismoran** [comment]
I been using version 5.1.6. almost 2 weeks and I can confirm that now it's working fine. Thanks, you have a great team.
